src/Engine/Spells/CastSpellInfo.cpp

```diff
@@ -1809,21 +1752,16 @@ void CastSpellInfoHelpers::castSpell() {
                             assert(false);
                     }
 
-                    if (spell_mastery == PLAYER_SKILL_MASTERY_NOVICE || spell_mastery == PLAYER_SKILL_MASTERY_EXPERT) {
-                        spell_fx_renderer->SetPlayerBuffAnim(
-                                pCastSpell->uSpellID, pCastSpell->uPlayerID_2);
-                        pParty->pPlayers[pCastSpell->uPlayerID_2]
-                            .pPlayerBuffs[PLAYER_BUFF_PRESERVATION]
-                            .Apply(pParty->GetPlayingTime() + spell_duration,
-                                   spell_mastery, 0, 0, 0);
+                    if (spell_mastery == PLAYER_SKILL_MASTERY_NOVICE ||
+                            spell_mastery == PLAYER_SKILL_MASTERY_EXPERT) {
```

Comment:
Btw, why are you breaking these up into two lines? We are pretty comfy with lines 120chars long.